app/code/Magento/Sales/Test/Mftf/ActionGroup/AdminClickRefundOfflineOnMemoDetailPageActionGroup.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminClickRefundOfflineOnMemoDetailPageActionGroup">
```

Comment:
Please, use the original `CreditMemo` entity name. Since just `Memo` might be confusing (it has many other meanings). 

```suggestion
    <actionGroup name="AdminClickRefundOfflineOnCreditMemoDetailPageActionGroup">
```

Thanks.

Reply:
Has been refactored.